docker-compose.yaml

```diff
@@ -11,6 +11,10 @@ services:
       - ./pg_data:/var/lib/postgresql/data
     environment:
       POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_USER: test
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: myapp
```

Comment:
are the `POSTGRES` parameters needed? have you tried adding only `user: postgres` ?

Reply:
user: postgres is there under environment, i don't know if the parameters can change

Reply:
Running it with only user:postgres seems to work as well, I would need to test it a bit more to be sure